src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/XamlObjectReader.cs

```diff
@@ -190,11 +185,7 @@ public virtual object Instance
         {
             get
             {
-                if (this.currentXamlNode.NodeType == XamlNodeType.StartObject)
-                {
-                    return this.currentInstance;
-                }
-                return null;
+                return this.currentXamlNode.NodeType == XamlNodeType.StartObject ? this.currentInstance : null;
```

Comment:
I'd change it for easier readability (maybe too personal view point?)
```suggestion
                return (this.currentXamlNode.NodeType == XamlNodeType.StartObject) ? this.currentInstance : null;
```

Reply:
Adding additional "(" only adds noise, IMHO.